README.md

```diff
@@ -28,8 +28,7 @@ You can configure linter-slim by editing ~/.atom/config.cson (choose Open Your C
 
 ```
```

Comment:
This section should be re-written to a general description of the settings. We've had _many_ problems with people manually editing the settings file and doing it wrong because they didn't know how it was supposed to work. Anyone who would know how it is supposed to work will already know how to edit it in there.

Something like this:
```markdown
Settings can be configured from Atom's settings page in the packages section.

Available settings are:
* **Slimrb Executable Path**: The _full_ path to `slimrb`, for example `/usr/bin/slimrb`. Leave at the default to use the system `$PATH` to find it
* **Rails**: Whether or not to generate Rails compatible code
* **Library**: Comma separated list of libraries or plugins to load when linting
```